core/src/main/java/com/scalar/db/sql/Example.java

```diff
@@ -0,0 +1,289 @@
+package com.scalar.db.sql;
+
+import com.scalar.db.sql.builder.StatementBuilder;
+import com.scalar.db.sql.exception.SqlException;
+
+public class Example {
+
+  private static final String NAMESPACE_NAME = "ns";
+  private static final String TABLE_NAME = "tbl";
+  private static final String COLUMN_NAME_1 = "col1";
+  private static final String COLUMN_NAME_2 = "col2";
+  private static final String COLUMN_NAME_3 = "col3";
+  private static final String COLUMN_NAME_4 = "col4";
+
+  public static void main(String[] args) {
+    try (SessionFactory sessionFactory =
+        SessionFactory.builder()
+            .withProperty("scalar.db.contact_points", "jdbc:mysql://localhost:3306/")
+            .withProperty("scalar.db.username", "root")
+            .withProperty("scalar.db.password", "mysql")
+            .withProperty("scalar.db.storage", "jdbc")
+            .build()) {
+      transactionModeExample(sessionFactory);
+      twoPhaseCommitTransactionModeExample(sessionFactory);
+    }
+  }
+
+  public static void transactionModeExample(SessionFactory sessionFactory) {
+    System.out.println("Start transaction mode example");
+
+    Session session = sessionFactory.getTransactionSession();
```

Comment:
We can get a session for transaction mode from `SessionFactory.getTransactionSession()`. And we can repeatedly reuse this session after committing or rolling back a transaction.